src/components/EventItemAuthor.jsx

```diff
@@ -0,0 +1,14 @@
+import { requestUser } from "@/lib/fetchAPI";
+
+export const EventItemAuthor = async ({ userid }) => {
+  let data;
+  try {
+    data = await requestUser(userid);
+  } catch (error) {
+    console.error("An error occurred while fetching the user:", error);
+    return;
+  }
+
+  let index = 0;
+  return <p>BY: {data ? data[index].name : "Error fetching user"}</p>;
```

Comment:
you can just directly return the string, so the <p> is in the parent component.

Therefore the style is determined by the parent
```tsx
<p>
  <EventItemAuthor ... />
</p>
```